test/AutoRest.TestServer.Tests/Mgmt/OutputLibrary/OutputLibraryTestBase.cs

```diff
@@ -31,5 +38,19 @@ protected static async Task<(CodeModel Model, BuildContext<MgmtOutputLibrary> Co
             _ = context.Library; // gen lib
             return (model, context);
         }
+
+        [Test]
+        public void ValidateResourceClassTypesCount()
```

Comment:
If this is the base class for the "general project as part of the test" why is there a test on it directly as well?

I don't have a good handle of the difference here.

Reply:
We want to run this test for all the classes inheriting this class. This test is just checking number of resource type classes are equals or not.

Reply:
This is a test that should pass for ever mgmt test project so adding it to the base ensures that it runs for all of them